ai-telemetry/base/zookeeper/statefulset.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: zookeeper
+  annotations:
+    template.alpha.openshift.io/wait-for-ready: "true"
+  labels:
+    app.kubernetes.io/instance: zookeeper
+    app.kubernetes.io/name: zookeeper
+    app.kubernetes.io/component: zookeeper
+    role: zookeeper
+spec:
+  replicas: 1
```

Comment:
Let's try scaling the replicas here to 3 for our cluster to test out high availability.